README.md

```diff
@@ -1038,13 +1034,13 @@ You can run individual tests in the `test` directory.  **Caution:** If you build
 
 ```bash
 $ cd <exiv2dir>/build
```

Comment:
Remove `/build`

Reply:
Removed in 34fddc8f5d808893b358f829d92b70a9c282ec05